server/notification-providers/opsgenie.js

```diff
@@ -54,7 +54,7 @@ class Opsgenie extends NotificationProvider {
             }
 
             if (heartbeatJSON.status === UP) {
-                let opsgenieAlertsCloseUrl = `${opsgenieAlertsUrl}/${encodeURIComponent(monitorJSON.name)}/close?identifierType=alias`;
+                let opsgenieAlertsCloseUrl = `${opsgenieAlertsUrl}/${encodeURIComponent(monitorJSON.pathName)}/close?identifierType=alias`;
```

Comment:
Seems risky without testing

```suggestion
                let opsgenieAlertsCloseUrl = `${opsgenieAlertsUrl}/${encodeURIComponent(monitorJSON.name)}/close?identifierType=alias`;
```